arbitrum-client/src/client/contract_interaction.rs

```diff
@@ -107,19 +120,34 @@ impl ArbitrumClient {
     /// `VALID WALLET UPDATE` statement
     ///
     /// Awaits until the transaction is confirmed on-chain
-    pub async fn update_wallet(
+    pub async fn update_wallet<
+        const MAX_BALANCES: usize,
+        const MAX_ORDERS: usize,
+        const MAX_FEES: usize,
+        const MERKLE_HEIGHT: usize,
```

Comment:
Same here with the generics and on all interfaces, each bundle type has an associated sized, boxed type